internals/cli/cmd_notice_test.go

```diff
@@ -36,6 +36,7 @@ func (s *PebbleSuite) TestNoticeID(c *C) {
 				"id": "123",
 				"type": "custom",
 				"key": "a.b/c",
+                "user-id": 1000,
```

Comment:
Looks like there may be a tabs/spaces issue here. I'll amend the last commit (haven't figured out how to suggest changes on a PR owned by me...)